neutrons/addie
# New column that allows to enter Align and Focus arguments

To test this PR
* create from scratch a new table (new rows) and change the last column values
* save the new table (json file)
* reload the config file created
json file should contain the new key/values arguments
